cli/src/program_v4.rs

```diff
@@ -300,16 +308,16 @@ impl ProgramV4SubCommands for App<'_, '_> {
                     SubCommand::with_name("show")
                         .about("Display information about a buffer or program")
                         .arg(
-                            Arg::with_name("account")
-                                .index(1)
```

Comment:
Do you know why this positional argument was specified? And, why is it ok to remove it now?

Reply:
It i not removed, just turned into a named argument `--program-id`, to be consistent with all other subcommands.